Bottom: 9df705152763bbb7fb6f626019a3630e1eafac40
Top:    daf36e1f7aa340a6627ee8ff336e6ebdfef94bba
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-24 11:22:46 -0400

Refresh of mmc-spi

---

diff --git a/sound/soc/fsl/efika-audio-fabric.c b/sound/soc/fsl/efika-audio-fabric.c
index 5126a81..b93e3b9 100644
--- a/sound/soc/fsl/efika-audio-fabric.c
+++ b/sound/soc/fsl/efika-audio-fabric.c
@@ -89,6 +89,6 @@ module_exit(efika_fabric_exit);
 
 
 MODULE_AUTHOR("Jon Smirl <jonsmirl@gmail.com>");
-MODULE_DESCRIPTION(DRV_NAME ": mpc5200 Efika fabric driver");
+MODULE_DESCRIPTION("mpc5200 Efika fabric driver");
 MODULE_LICENSE("GPL");
 
diff --git a/sound/soc/fsl/pcm030-audio-fabric.c b/sound/soc/fsl/pcm030-audio-fabric.c
index 4bd957a..0a50bbd 100644
--- a/sound/soc/fsl/pcm030-audio-fabric.c
+++ b/sound/soc/fsl/pcm030-audio-fabric.c
@@ -89,6 +89,6 @@ module_exit(pcm030_fabric_exit);
 
 
 MODULE_AUTHOR("Jon Smirl <jonsmirl@gmail.com>");
-MODULE_DESCRIPTION(DRV_NAME ": mpc5200 pcm030 fabric driver");
+MODULE_DESCRIPTION("mpc5200 pcm030 fabric driver");
 MODULE_LICENSE("GPL");
